docs/plugins.md

```diff
@@ -40,8 +38,9 @@ exports.config = {
 };
 ```
 
-Protractor contains built in plugins in the 'plugins' folder. An example of
-using the 'ngHint' plugin is shown below.
+Protractor's default [Plugins](/docs/plugins.md#first-party-plugins) have 
```

Comment:
I don't think this sentence is helpful at all anymore. Let's just kill it and say "An example of using the `ngHint` plugin is shown below"